docs/guides/connecting_to_your_data/overview.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Connect to data: Overview"
+---
+# [![Connect to data icon](../../images/universal_map/Outlet-active.png)](overview.md) Connect to data: Overview 
+
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<!--Use 'inactive' or 'active' to indicate which Universal Map steps this term has a use case within.-->
+
+<UniversalMap setup='inactive' connect='active' create='inactive' validate='inactive'/>
+
+<!-- Only keep one of the 'To best understand this document' lines.  For processes like the Universal Map steps, use the first one.  For processes like the Architecture Reviews, use the second one. -->
+
+:::note Prerequisites
+- Completing [Step 2: Connect to data](/docs/tutorials/getting_started/connect_to_data) of the Getting Started tutorial is recommended.
+:::
+	
+Connecting to your data in Great Expectations is designed to be a painless process.  Once you have performed this step, you will have a consistent API for accessing and validating data on all kinds of source data systems: SQL-type data sources, local and remote file stores, in-memory data frames, and more.
+
+## The connect to data process
+
+<!-- Brief outline of what the process entails.  -->
+
+Connecting to your data is built around the Datasource object.  A Datasource provides a standard API for accessing and interacting with data from a wide variety of source systems.  It does this by providing an interface for a Data Connector and an Execution Engine to work together.
+
+The majority of the work involved in connecting to data is a simple matter of configuring a new Datasource according to the requirements of your underlying data system.  Once your Datasource is configured and saved to your Data Context you will only need to use the Datasource API to access and interact with your data, regardless of the original source system (or systems) that your data is stored in.
+
+<!-- The following subsections should be repeated as necessary.  They should give a high level map of the things that need to be done or optionally can be done in this process, preferably in the order that they should be addressed (assuming there is one). If the process crosses multiple steps of the Universal Map, use the <SetupHeader> <ConnectHeader> <CreateHeader> and <ValidateHeader> tags to indicate which Universal Map step the subsections fall under. -->
+
+### 1. Boilerplate
```

Comment:
Is this the header we want?

Reply:
I'm not really committed to it.  Previously it was "Create your datasource" -- except we aren't really doing that through the CLI.  What we're doing is creating the boilerplate that will follow along with the steps after this.  I am open to renaming this section.